farnaz/challenge7.html

```diff
@@ -0,0 +1,192 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <script>
+        class Warehouse {
```

Comment:
Excellent idea of using `class`. Could you please explain what are the benefits and why have you used this approach?

Reply:
This helps us to have fewer errors, simpler syntax, easier to call prototypes and helps to organize the program more and etc.